src/api/sysinfo.go

```diff
@@ -0,0 +1,70 @@
+package api
+
+import (
+	"github.com/zcalusic/sysinfo"
+	"github.com/shirou/gopsutil/v3/process"
+	"strings"
+)
+
+/* TYPES */
+
+type Distro struct {
+	Name    string
+	Version string
+}
+
+type InitSystem string
+
+func (is InitSystem) String() string {
```

Comment:
It's generally a good idea to avoid acronyms unless they are very well establish. In this case I think the best name for the variable is `initSystem`

Reply:
I disagree
`String` is method on struct InitSystem, where `is` is instance of this type. A lot of people in go use short acronyms in such situation. Even JetBrains GoLand IDE often suggest short 1 char variables.